extension/src/experiments/columns/collect/index.ts

```diff
@@ -28,38 +28,50 @@ const collectFromExperiment = (
 
   const { data } = experiment
   if (data) {
-    collectMetricsAndParams(acc, data)
-    collectDeps(acc, data)
+    return Promise.all([
+      collectMetricsAndParams(acc, data),
+      collectDeps(acc, data)
+    ])
   }
 }
 
-const collectFromExperiments = (
+const collectFromExperiments = async (
   acc: ColumnAccumulator,
   experiments?: ExpRange[] | null
 ) => {
   if (!experiments) {
     return
   }
+
+  const promises = []
   for (const { revs } of experiments) {
-    collectFromExperiment(acc, revs[0])
+    promises.push(collectFromExperiment(acc, revs[0]))
   }
+
+  await Promise.all(promises)
```

Comment:
[F] I am now going to turn on https://eslint.org/docs/latest/rules/no-await-in-loop in the next PR so get ready to see this pattern a lot

Reply:
☝🏻 actually didn't end up turning it on but did use it to identify issues